net/tshttpproxy/tshttpproxy.go

```diff
@@ -38,6 +39,10 @@ var (
 	proxyFunc    func(*url.URL) (*url.URL, error)
 )
 
+func init() {
+	envknob.ApplyDiskConfig()
+}
```

Comment:
This isn't the right time or place to do this.

Init-time work should generally be avoided, and ApplyDiskConfig in particular needs to be done at a controlled time. This would mean we now do that ApplyDiskConfig twice in tailscaled: once during init before main, and once in main.

ipn-go-bridge also already calls this, in RunBackend.

If you need it earlier than that, it seems like the fix is in ipn-go-bridge instead.